<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflSaveTabWidget</class>
  <widget class="QWidget" name="ReflSaveTabWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>959</width>
        <height>346</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Save ASCII</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
        <number>20</number>
      </property>
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QLabel" name="savePathLabel">
              <property name="text">
                <string>Save path:</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="savePathEdit"/>
          </item>
          <item>
            <widget class="QLabel" name="prefixLabel">
              <property name="text">
                <string>Prefix:</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="prefixEdit"/>
          </item>
          <item>
            <spacer name="prefixSpacer">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QGroupBox" name="customSaveGroup">
          <property name="title">
            <string>Custom save</string>
          </property>
          <layout class="QGridLayout" name="customSaveLayout">
            <property name="margin">
              <number>20</number>
            </property>
            <item row ="0" column="0">
            <widget class="QLabel" name="filterLabel">
              <property name="text">
                <string>Filter</string>
              </property>
            </widget>
            </item>
            <item row="0" column="1">
              <widget class="QLineEdit" name="filterEdit"/>
            </item>
            <item row="0" column="2">
              <widget class="QCheckBox" name="regexCheckBox"/>
            </item>
            <item row="0" column="3">
              <widget class="QLabel" name="regexLabel">
                <property name="text">
                  <string>Regex</string>
                </property>
              </widget>
            </item>
            <item row="0" column="4">
              <spacer name="customSaveSpacer0">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item row ="1" column="0" colspan="4">
              <widget class="QLabel" name="listOfWorkspacesLabel">
                <property name="text">
                  <string>List Of Workspaces</string>
                </property>
              </widget>
            </item>
            <item row="1" column="4">
              <spacer name="customSaveSpacer1">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item row ="1" column="5">
              <widget class="QLabel" name="listOfLoggedParametersLabel">
                <property name="text">
                  <string>List Of Logged Parameters</string>
                </property>
              </widget>
            </item>
            <item row ="2" column="0" colspan="4">
              <widget class="QListWidget" name="listOfWorkspaces">
                <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
              </widget>
            </item>
            <item row="2" column="4">
              <spacer name="customSaveSpacer2">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item row ="2" column="5">
              <widget class="QListWidget" name="listOfLoggedParameters">
                <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
              </widget>
            </item>
            <item row="3" column="0" colspan="4">
              <widget class="QPushButton" name="refreshButton">
                <property name="text">
                  <string>Refresh</string>
                </property>
              </widget>
            </item>
            <item row="3" column="4">
              <spacer name="customSaveSpacer3">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item row="3" column="5" rowspan="4">
              <widget class="QGroupBox" name="customFormatOptionsGroup">
                <property name="title">
                  <string>Custom Format Options</string>
                </property>
                <layout class="QGridLayout" name="customFormatOptionsLayout">
                  <item row="0" column="0">
                    <widget class="QCheckBox" name="titleCheckBox"/>
                  </item>
                  <item row="0" column="1">
                    <widget class="QLabel" name="titleLabel">
                      <property name="text">
                        <string>Title</string>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="2">
                    <widget class="QCheckBox" name="qResolutionCheckBox"/>
                  </item>
                  <item row="0" column="3">
                    <widget class="QLabel" name="qResolutionLabel">
                      <property name="text">
                        <string>Q resolution</string>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="4">
                    <spacer name="customFormatOptionsSpacer0">
                      <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                        <size>
                          <width>20</width>
                          <height>20</height>
                        </size>
                      </property>
                    </spacer>
                  </item>
                  <item row="1" column="0" colspan="5">
                    <widget class="QGroupBox" name="separatorGroup">
                      <property name="title">
                        <string>Separator</string>
                      </property>
                      <layout class="QGridLayout" name="separatorLayout">
                        <item row="0" column="0">
                          <widget class="QRadioButton" name="commaRadioButton">
                            <property name="text">
                              <string>Comma</string>
                            </property>
                            <property name="checked">
                              <bool>true</bool>
                            </property>
                            <attribute name="buttonGroup">
                              <string notr="true">separatorButtonGroup</string>
                            </attribute>
                          </widget>
                        </item>
                        <item row="0" column="1">
                          <widget class="QRadioButton" name="spaceRadioButton">
                            <property name="text">
                              <string>Space</string>
                            </property>
                            <attribute name="buttonGroup">
                              <string notr="true">separatorButtonGroup</string>
                            </attribute>
                          </widget>
                        </item>
                        <item row="0" column="2">
                          <widget class="QRadioButton" name="tabRadioButton">
                            <property name="text">
                              <string>Tab</string>
                            </property>
                            <attribute name="buttonGroup">
                              <string notr="true">separatorButtonGroup</string>
                            </attribute>
                          </widget>
                        </item>
                        <item row="0" column="3">
                          <spacer name="separatorSpacer0">
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                              <size>
                                <width>20</width>
                                <height>20</height>
                              </size>
                            </property>
                          </spacer>
                        </item>
                      </layout>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>     
            <item row ="5" column="0">
              <widget class="QLabel" name="fileFormatLabel">
                <property name="text">
                  <string>File format</string>
                </property>
              </widget>
            </item>
            <item row="5" column="1" colspan="3">
              <widget class="QComboBox" name="fileFormatComboBox">
                <item>
                  <property name="text">
                    <string>Custom format (*.dat)</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>3 column (*.dat)</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>ANSTO, MotoFit, 4 column (*.txt)</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>ILL Cosmos (*.mft)</string>
                  </property>
                </item>
                <property name="font">
                  <font>
                    <bold>true</bold>
                  </font>
                </property>
              </widget>
            </item>
            <item row="5" column="4">
              <spacer name="customSaveSpacer5">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
            <item row="6" column="0" colspan="4">
              <widget class="QPushButton" name="saveButton">
                <property name="text">
                  <string>Save</string>
                </property>
              </widget>
            </item>
            <item row="6" column="4">
              <spacer name="customSaveSpacer6">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>20</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>    
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
 <tabstops/>
 <customwidgets/>
 <resources/>
 <connections/>
<buttongroups>
  <buttongroup name="separatorButtonGroup"/>
</buttongroups>
</ui>
